markdown/articles/2021/07/MonkeyWorld.md

```diff
@@ -0,0 +1,193 @@
+4 {{Brainstorm}}
+4 {{Ponder}}
+0-3 {{Preordain}} (lists with higher counts of Preordain try to improve their Bant Miracles matchup)
+2-4 {{Personal Tutor}}
+2-4 Discard Spells (2-4 {{Duress}} + 0-2 {{Thoughtsieze}})
+4 {{Dark Ritual}}
+4 {{Daze}}
+1-2 {{Edge of Autumn}} (more Cyclers help with the consistency of same turn wins)
+1 {{Ideas Unbound}}
+1-2 {{Thassa’s Oracle}} (2 Oracles help with Endurance)
+4 {{Doomsday}}
+2-4 {{Street Wraith}} (again more Cyclers help with the consistency of same turn wins)
+4 {{Force of Will}}
+
+Sideboard
+
+3-4 {{Force of Negation}} (a very important tool against other combo decks or prison decks)
+0-1 {{Pact of Negation}}
+1 {{Massacre}}
+0-2 Discard Spells
+2-4 One Mana Counterspells like {{Pyroblast}}, {{Flusterstorm}} or {{Mystical Dispute}}
+0-2 Cheap Creature Removal {{Fatal Push}}
+0-2 Bounce Spells like {{Echoing Truth}}
+1 Anti-Endurance Package ({{Shelldock Isle}} + {{Emrakul the Aeon’s Torn}} or something like {{Echo of Eons}})
+
+
+The most important changes post MH2 are the packages to beat {{Endurance}} and the absence of {{Defense Grid}}.
+
+Many Players moved away from {{Defense Grid}} and added 1-2 {{Pact of Negation}} to their lists.
+{{Defense Grid}} got a lot worse in the now faster Delver Matchups and {{Pact of Negation}} supports the new gameplan better (and also helps with {{Endurance}}).
+It doesn’t need extra resources like mana or pitch cards and really helps winning in one turn.
+
+The most important techs emerged to beat {{Endurance}}.
+Some lists play two {{Thassa's Oracle}} or {{Echo of Eons}}, but the most common techs are {{Shelldock Isle}} + {{Emrakul, the Aeon’s Torn}} and {{Pact of Negation}}.
+The Emrakul plan can even be combined with Thassa's Oracle for very resilient piles, but be careful you can’t activate Shelldock Isle when your opponent has a {{Teferi, Time Raveler}} and {{Back to Basics}} is also very nasty.
+Pact of Negation is not only a good card against Delver, but also really helps beating {{Endurance}}.
+
+
+Last but not least there also some cute Lists with very special takes on Doomsday.
+WonderPreaux got some good results with a {{Stifle}} + {{Wasteland}} tech and players like Enrichetta developed a slower version of UB Doomsday to beat Delver from a more controlling angle.
+There are even some lists out there with {{Watery Grave}} as their 5th {{Underground Sea}}.
+
+## Conclusion
+
+Overall we lost some percentage points against many decks we had really good matchups against.
+In most we still are very favored, but they are no longer “free wins”.
+In my opinion Doomsday is still the best Combo deck out there, but not as dominant against every non-Delver Deck as it was pre MH2.
+I hope after this insight into the changes to the metagame and our deck, you feel prepared for the months coming.
+
+Should you still have questions, don’t shy away from messaging me on Discord or Twitter.
+
```

Comment:
Please add the following generic article sign off text :)

: 
```
As always if you want even more content please check out our
[resources](/appendices/resources/) page which includes a list of community
streamers or maybe if you are up for a challenge maybe check out the
[puzzles](/puzzles/) page.

If you would like to submit any content to the Wiki then please contact
AngryBacon in the [Doomsday Discord](https://discord.gg/vajvFXt) or contact
Doishy on the Discord if you wish to discuss anything mentioned in the article
further.
```

Reply:
If we want that in all articles, I could add it by default so that we don't have to write it every time. Would that be better?

Reply:
> 
> 
> If we want that in all articles, I could add it by default so that we don't have to write it every time. Would that be better?

Yeah maybe